Sources/SwiftProtobuf/Google_Protobuf_Any+Extensions.swift

```diff
@@ -152,10 +152,16 @@ extension Google_Protobuf_Any: _CustomJSONCodable {
     } else {
       // This is not using the specialized encoding, so we can use the
       // standard path to decode the binary value.
+      // First, clear the fields so we don't waste time re-serializing
+      // the previous contents if a field gets repeated.
```

Comment:
I'd probably expand this comment a little, this makes it sorta sound like it impacts a repeated Any field.  Instead you are trying to handle the case where the field name/number was repeated but is actually a single field, so the second occurrence will replace the first.

Reply:
I figured out how to just edit it in place.